themes/bootstrap3/js/lightbox.js

```diff
@@ -131,21 +134,25 @@ VuFind.register('lightbox', function Lightbox() {
           VuFind.refreshPage();
           return;
         }
+        var testDiv = $('<div/>').html(content);
+        var errorMsgs = testDiv.find('.flash-message.alert-danger:not([data-lightbox-ignore])');
         // Place Hold error isolation
-        if (obj.url.match(/\/Record/) && (obj.url.match(/Hold\?/) || obj.url.match(/Request\?/))) {
-          var testDiv = $('<div/>').html(content);
-          var error = testDiv.find('.flash-message.alert-danger');
-          if (error.length && testDiv.find('.record').length) {
-            showAlert(error[0].innerHTML, 'danger');
+        if (obj.url.match(/\/(?!Record|Hold|Request)\?/)) {
```

Comment:
I think something is wrong here. The condition `obj.url.match(/\/Record/) && (obj.url.match(/Hold\?/) || obj.url.match(/Request\?/))` has been changed to `obj.url.match(/\/(?!Record|Hold|Request)\?/)`, which is drastically different in meaning -- it flattens out a two-part condition and changes positive matching to negative matching. Don't we actually want something more like `/\/Record\/.*(Hold|Request)\?/` to match the intent of the original regex?